nvflare/fuel/utils/pipe/pipe_handler.py

```diff
@@ -109,7 +110,7 @@ def __init__(
         self.msg_cb = None
         self.msg_cb_args = None
         self.msg_cb_kwargs = None
-        self._other_end_is_up = threading.Event()
+        self.other_end_is_up_or_dead = threading.Event()
```

Comment:
Rename to "peer_is_up_or_dead" since we have been using "peer" to mean "the other end".